table/table_properties.cc

```diff
@@ -71,6 +71,9 @@ std::string TableProperties::ToString(
       filter_policy_name.empty() ? std::string("N/A") : filter_policy_name,
       prop_delim, kv_delim);
 
+  AppendProperty(result, "compression",
+                 compression_type, prop_delim, kv_delim);
```

Comment:
Since this is a debug code-path, it might be useful to convert the compression type to string here. currently this will output compression type 0, 1, 2, etc. it'll be better if we actually dump "no_compression", "snappy", etc...